tests/test_pytest/test_plugin_schema/test_product.yaml

```diff
@@ -0,0 +1,13 @@
+interface: product
```

Comment:
is this the same as good/products/single.yaml ?  Or some other product test?  This seems like I've seen it before.

Reply:
Deleted.